grails-app/domain/mama/ng/scheduler/User.groovy

```diff
@@ -22,4 +22,8 @@ class User {
         GENERAL,
         SUPER
     }
+
+    boolean checkPassword(String password) {
+        return (password.bytes.encodeBase64().toString().equals(passwordHash))
```

Comment:
Is there something simple we can do so that passwords are not stored as plain text?